llvm/test/Transforms/LoopVectorize/RISCV/vplan-vp-intrinsics-reduction.ll

```diff
@@ -93,6 +94,7 @@ define i32 @reduction(ptr %a, i64 %n, i32 %start) {
 ; NO-VP-OUTLOOP-EMPTY:
 ; NO-VP-OUTLOOP-NEXT: middle.block:
 ; NO-VP-OUTLOOP-NEXT:   EMIT vp<[[RDX:%.+]]> = compute-reduction-result ir<[[RDX_PHI]]>, ir<[[ADD]]>
+; NO-VP-OUTLOOP-NEXT:   EMIT vp<[[RDX_EX:%.+]]> = extract-from-end vp<[[RDX]]>, ir<1>
```

Comment:
redundant extracts can be folded by VPlan-transform as follow-up

Reply:
fine, is this extract redundant?

Reply:
Curious to identify redundant extracts.